FFXIVClientStructs/FFXIV/Client/Game/GcArmyManager.cs

```diff
@@ -93,10 +95,12 @@ public unsafe partial struct GcArmyMember {
     [FieldOffset(0x48)] public byte StainHands;
     [FieldOffset(0x49)] public byte StainLegs;
     [FieldOffset(0x4A)] public byte StainFeet;
-    [FieldOffset(0x4B)] public byte MasteryIndependent;
-    [FieldOffset(0x4C)] public byte MasteryOffensive;
-    [FieldOffset(0x4D)] public byte MasteryDefensive;
-    [FieldOffset(0x4E)] public byte MasteryBalanced;
+    // TODO: Verify offsets above
+
```

Comment:
Cool, now I can suggest this here. :joy:
The other fields are still there. They just added a second row of stain ids for the new dye channels.

```suggestion
    [FieldOffset(0x4B)] public byte Stain2MainHand;
    [FieldOffset(0x4C)] public byte Stain2OffHand;
    [FieldOffset(0x4D)] public byte Stain2Head;
    [FieldOffset(0x4E)] public byte Stain2Body;
    [FieldOffset(0x4F)] public byte Stain2Hands;
    [FieldOffset(0x50)] public byte Stain2Legs;
    [FieldOffset(0x51)] public byte Stain2Feet;
```